### ISSUE TITLE
Fold/unfold all doesn't work?

### ISSUE THREAD

**Issue by alphapapa** [orgzly/orgzly-android#10]
Hi,

I just installed and started using Orgzly with my existing Org notes in Dropbox.  It seems to work great, but the fold/unfold-all button doesn't seem to do anything.  No matter how many times I tap it, or how long I wait, or how small a file I try it on, it doesn't do anything--so I have to collapse everything manually, which is obviously a chore in my large Org files.  :)

Thanks for your work!

**Comment by nevenz** [comment]
Yeah, this is a big one and main reason why #8 (auto sync) isn't yet implemented.

It's not just fold/unfold, anything that needs to write to db will be queued.
